feature-libs/organization/administration/occ/converters/occ-permission-type-list.normalizer.ts

```diff
@@ -23,10 +23,12 @@ export class OccPermissionTypeListNormalizer
     target?: OrderApprovalPermissionType[]
   ): OrderApprovalPermissionType[] {
     if (target === undefined) {
-      target = source.orderApprovalPermissionTypes.map((permissionType) =>
-        this.converter.convert(permissionType, PERMISSION_TYPE_NORMALIZER)
-      );
+      // target = what should be here??
```

Comment:
@Platonn

Reply:
```ts
      target = { ...(source as any) };
```

Reply:
still I don't see sense, in next line it will be overwritten anyway

Reply:
My bad, I didn't take next line into account and that it's conversion into array.

Agreed. Above fallback object initialization doesn't make sense in this case.